### DISCUSSION TITLE
Replace Nzbhydra with Nzbhydra2

### DISCUSSION THREAD

**Discussion by ghost** [swizzin/swizzin#157] [Feature Requests] [answered]
I managed to do this manually
by installing nzbhydra via the commandline, and then migrating to nzbhydra2  via the built in migration tool. 
- this allows reverse proxy and domain name to work, otherwise I could not get it to detect and would just bring a blank page

- The downside is that the link is incorrect in the panel, so the url has to be access manually, also I have to keep nzbhydra1 install

Saying that having this package updating would bring a number of fixes that the second verison has over the first.

**Comment by brodonalds** [comment]
seconded here.

**Comment by flying-sausages** [comment]
> I managed to do this manually

If you can share your steps I'm sure others would find it helpful, and would help whoever will implement this.

**Comment by bakerboy448** [comment, marked as answer]
complete in b204df5676eb80935cbd08596824969eb44c3483
